bin/bridge-client/src/xpriv.rs

```diff
@@ -0,0 +1,51 @@
+//! Parses the operator's master xpriv from a file.
+
+use std::{
+    env,
+    path::{Path, PathBuf},
+};
+
+use bitcoin::bip32::Xpriv;
+use strata_key_derivation::operator::OperatorKeys;
+use strata_primitives::keys::ZeroizableXpriv;
+use zeroize::Zeroize;
+
+/// The environment variable that contains the operator's master [`Xpriv`].
+const OPXPRIV_ENVVAR: &str = "STRATA_OP_MASTER_XPRIV";
+
+/// Parses the master [`Xpriv`] from a file.
+pub(crate) fn parse_master_xpriv(path: &Path) -> anyhow::Result<OperatorKeys> {
+    let mut xpriv_str = std::fs::read_to_string(path)?;
+    let xpriv: ZeroizableXpriv = xpriv_str.parse::<Xpriv>()?.into();
+
+    // Zeroize the buffer
+    xpriv_str.zeroize();
```

Comment:
If the parsing of `xpriv` fails, the early return will not zeroize `xpriv_str`. You could argue that if this happens, `xpriv_str` doesn't represent a "valid" secret value that needs zeroizing; otherwise, a `Zeroizing` wrapper will handle it.

I favor the latter, but only slightly. If you go with the former, I'd make a note about this in the comment.

Reply:
What if we don't use a str in the first place, see 2946fdf2373fe1eb3cd3c458f52e5f9a0a83c9a0

Reply:
There's still a string hanging around in memory for that approach too. Zeroizing is super duper annoying :)

Reply:
Oooo Ok let me try again in afaa63847b349afd78671b673a04e6d5e4396059